### ISSUE TITLE
[WFRP4e Core] Prayers

### ISSUE THREAD

**Issue by Txus5012** [moo-man/WFRP4e-FoundryVTT#790]
Thanks for reading the post about Prayers I wrote earlier. Here are a few issues and an idea.

Blessings:
-Blessing of Healing and Blessing of Recuperation: Now they have range Touch. The only thing it needs now to be perfect is the ability to make them ranged with overcasts. That can be easily achieved with these settings.
![Blessings Overcast](https://user-images.githubusercontent.com/87753744/129910474-218672c1-87ed-4721-8494-2650cd755a2f.jpg)
-Blessings of Charisma and Finesse have empty Effect Value.

Prayers:
-Anchorite's Endurance shows an error message when applied to a token with conditions that cause penalties (poisoned, fatigued, etc.)
REPORT
EFFECT:	Anchorite's Endurance
ACTOR:	Dummy - TPjTXmWdVGCweiqH
ERROR:	TypeError: Cannot set property flags of #<EffectWfrp4e> which has only a getter

-Balm to a Wounded Mind isn't applied. There is a console error everytime the cursor hovers the token or the character sheet is opened:
| The effect Balm to a Wounded Mind threw an error when being prepared. TypeError: this.parent.items.get is not a function
(I overlooked this one the first time)

One question:
Some effects in Spells and Prayers that add a value to a characteristic, also removes a value to it's bonus (e.g. Blessing of Courage, Hardiness). Is there a reason for that?

**Comment by moo-man** [comment]
Yes, it's to avoid changing the wound calculation

**Comment by Txus5012** [comment]
Changing the characteristic shouldn't actually change the wound calculation?

**Comment by moo-man** [comment]
Not for temporary bonuses like blessings, as clarified here https://www.cubicle7games.com/wfrp-faq/

**Comment by Txus5012** [comment]
Didn't remember that. OK. Thanks.

**Comment by moo-man** [comment]
I can't seem to reproduce the error with Anchorite's Endurance. The error shows up on application? or when they try to roll? Both don't happen for me.

**Comment by Txus5012** [comment]
Actually, there are two ways to reproduce it.
1) Add a poisoned condition to a token (also happens with deafened, stunned, entangled, fatigued, blinded, broken, prone, unconscious (probably just because it adds prone and fatigued) and grappling) and then apply the miracle.
2) Apply the miracle and then add one of those conditions.

**Comment by moo-man** [comment]
Neither of these produce an error, are you sure you're using the new prayer from the compendium? What is the script?

**Comment by Txus5012** [comment]
Yes, I'm using the new prayer.

Do you mean this?
![Anchorite's Endurance](https://user-images.githubusercontent.com/87753744/130270364-ad9e7159-e068-4e7c-91f9-5cd8f3628bff.jpg)

**Comment by moo-man** [comment]
Yeah that script is different from mine, I guess this wasn't included in the last update.
